Impossible-Agent-Suite/gas-control-agent/src/agent.ts

```diff
@@ -0,0 +1,27 @@
+import BigNumber from 'bignumber.js';
+import { Finding, HandleTransaction, TransactionEvent } from 'forta-agent';
+import { CONTRACTS, createFinding, hexToNumber } from './utils';
+
+export const provideHandleTransaction = (
+  contracts: string[]
+): HandleTransaction => {
+  return async (txEvent: TransactionEvent): Promise<Finding[]> => {
+    const findings: Finding[] = [];
+
+    const th = new BigNumber(10); // 10 GWei
+    const gasPrice = hexToNumber(txEvent.gasPrice);
+
+    contracts.map((contract) => {
+      if (txEvent.to === contract && gasPrice.gt(th)) {
+        findings.push(createFinding(contract, gasPrice.toString()));
+      }
+    });
```

Comment:
Consider changing to `foreach` because you are not mapping contracts

Reply:
range [7, 10)  is not necessary imo. Because they just want to flag >10